### PR TITLE
added linking for notification text

### PR DESCRIPTION
**Pull request by samandmoore**
- bundled css and js on notifications page
- moved js message linkification code to Chat.utility.js
- set up message linkification on notifications page

### CODE REVIEW

**Comment by davidfowl** [comment]
Room links are broken right? The hash links?

File: JabbR/Views/Notifications/index.cshtml [hunk 1]
```diff
@@ -64,7 +70,7 @@
                                         </a>
                                     }
                                     <blockquote>
-                                        <p>@notification.Message</p>
+                                        <p class="@(!notification.HtmlEncoded ? "linkify" : "")">@notification.Message</p>
```
Comment (davidfowl) [inline JabbR/Views/Notifications/index.cshtml:73]: Change to :

```
@(notification.HtmlEncoded ? "" : "linkify")
```

**Comment by davidfowl** [comment]
I merged it in

**Comment by samandmoore** [comment]
Not broken, disabled :wink:

**Pull request closed without merging**